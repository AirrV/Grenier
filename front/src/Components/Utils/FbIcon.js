import React, { Component, Fragment } from "react";
import { withStyles } from "@material-ui/core/styles";

let styles = {
  image: {
    height: "25px",
    opacity: "0.6"
  }
};

class FbIcon extends Component {
  render() {
    const { classes } = this.props;

    return (
      <Fragment>
        <img
          src="images/facebook.png"
          alt="facebook-logo"
          className={classes.image}
        />
      </Fragment>
    );
  }
}

export default withStyles(styles)(FbIcon);
