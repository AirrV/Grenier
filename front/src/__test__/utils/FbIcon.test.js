import React from 'react';
import renderer from "react-test-renderer";
import FbIcon from '../../components/Utils/FbIcon';

it("Renders the welcome page", () => {
  const tree = renderer
    .create(<FbIcon />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})