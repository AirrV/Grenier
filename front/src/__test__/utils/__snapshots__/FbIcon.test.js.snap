// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the welcome page 1`] = `
<img
  alt="facebook-logo"
  className="FbIcon-image-1"
  src="images/facebook.png"
/>
`;
